### ISSUE TITLE
Scout Report export

### ISSUE THREAD

**Issue by kkemp85** [legiongis/hmsf-md#232]
As of now, the export of Scout Reports results in a jumble of columns that don't follow the order of the Scout Report itself. It would be good if we could have these export in the exact order they are on the live reports in Arches. This may cause issues with land managers who are using the data for reporting.

<img width="799" alt="image" src="https://user-images.githubusercontent.com/54329725/150231822-f7c309ca-18bf-4ddf-b8fb-4414814b528c.png">

**Comment by mradamcox** [comment]
Once we upgrade to Arches 6.0 this will be fixed, so I'm going to put it "on hold" for now.

**Comment by mradamcox** [comment]
Reevaluate after #238 is closed.
